src/com/palmergames/bukkit/towny/command/InviteCommand.java

```diff
@@ -110,13 +111,13 @@ private static void parseInviteList(Player player, String[] split) {
 		Optional<Resident> resOpt = TownyUniverse.getInstance().getResidentOpt(player.getUniqueId());
 		
 		if (!resOpt.isPresent()) {
-			TownyMessaging.sendErrorMsg(player, Translation.of("msg_err_not_registered"));
+			TownyMessaging.sendErrorMsg(player, Translatable.of("msg_err_not_registered"));
 			return;
 		}
 		
 		Resident resident = resOpt.get();
 
-		String received = Translation.of("player_received_invites")
+		String received = Translation.of("player_received_invites", player)
```

Comment:
I think this was meant to be changed to Translatable.of(), as the lang string cannot accept an %s for the player.

Reply:
The player argument is there for getting the locale, could not make it a Translatable because of the .replace's below that line